src/views/tutorial-view/components/variants-dropdown-disclosure/variants-dropdown-disclosure.module.css

```diff
@@ -0,0 +1,25 @@
+.root {
+	margin-bottom: 24px;
+}
+
+.label {
+	margin-bottom: 8px;
+	color: var(--token-color-foreground-primary);
+}
+
+.dropdownDisclosure {
+	/** 
+	* 🏴‍☠️ FLAG
+	* Reaching into the component here to
+	* override styles. This would be better to
+	* update the actual dropdown disclosure list width, which
+	* is currently set to `max-content`.
+	*/
```

Comment:
I chose to not make the update on dropdown disclosure list directly in this PR because that would really increase the scope of review. I did a quick test and updating from `max-content` seemed fine but I just didn't want to balloon the changes here, so this could be a quick follow along clean up item 🧼

Reply:
🙌 Thanks for calling this out - this is ringing bells and I've seen something similar at least once - in [version-selector](https://github.com/hashicorp/dev-portal/blob/main/src/components/version-switcher/version-switcher.module.css) work recently, at least.

We have a follow-up task to try to resolve the duplicate "reach-in-ish" CSS at some point: [[discuss] Consider fullWidth prop for DropdownDisclosure to avoid style overrides](https://app.asana.com/0/1202097197789424/1204412156894162/f). Feels like this is another data point there 👍 Heck maybe this `fit-content` fix would be an alternative to the more complex CSS overrides we have in `version-selector` 🤔

Reply:
I can add a note about this case to that ticket! Good to hear you've ran into this as well.